### PR TITLE
Fixed - Label is overflowing.

### LINKED ISSUE #1073: [UI] Label is overflowing
STR:

1. Navigate to https://taskcluster-ui.herokuapp.com/
2. Click sign in
3. Set Client ID to "test", Access Token to "test" and JSON certificate to `111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111`
4. Navigate to https://taskcluster-ui.herokuapp.com/profile

Expected: The certificate field has its value wrap around.
Actual: The certificate field value doesn't wrap around.

We use material-ui for our component. You can probably look at https://material-ui.com/components/lists/#align-list-items and see how they wrap around the text there.

### PR DESCRIPTION
**Pull request by rbrishabh**
Fixes #1073
@helfi92 

Before : The certificate field value doesn't wrap around.

After: The certificate field has its value wrap around.
bug-2

Let me know if this needs any changes!
Thank you! :)

### CODE REVIEW

File: ui/src/views/Profile/index.jsx [hunk 1]
```diff
@@ -45,7 +45,9 @@ export default class Profile extends Component {
                   primary="Certificate"
                   secondary={
                     user.credentials.certificate ? (
-                      <code>{user.credentials.certificate}</code>
+                      <code style={{ wordBreak: 'break-word' }}>
+                        {user.credentials.certificate}
+                      </code>
```
Comment (helfi92) [inline ui/src/views/Profile/index.jsx]: Really nice! Could we use the `withStyles` decorator for the styling. Let's separate the jsx from the css. Here's an example: https://github.com/taskcluster/taskcluster/blob/ea9485f2f936ea0c115e26625f6fd2095030216e/ui/src/views/Hooks/ListHooks/index.jsx#L18


```jsx
import { withStyles } from '@material-ui/core/styles';

@withStyles({
  certificate: {
      wordBreak: 'break-word',
  },
})

// ....

<code className={classes.certificate}>
  {user.credentials.certificate}
</code>
```

Comment (rbrishabh) [reply ui/src/views/Profile/index.jsx]: Alright. Got it.

I am just not sure how we can access ```certificate``` using ```classes``` as we never really define it anywhere and we are importing the css with ```withStyles```. What am I missing?

Do I need to define ```classes``` somewhere?

Comment (helfi92) [reply ui/src/views/Profile/index.jsx]: The usage of `@withStyles` will inject `classes` in props.

Comment (rbrishabh) [reply ui/src/views/Profile/index.jsx]: Without writing the word ```classes``` explicitly in ```props```, I am getting the error 
> error: 'classes' is not defined (no-undef) at src\views\Profile\index.jsx:54:40:

But after I write `classes` in props myself, everything seems to be working okay.

I will commit the changes for you to review once.

**Comment by rbrishabh** [comment]
You are welcome!
@helfi92 Couldn't have done this without your help! You rock!

I am really enjoying this, so if I can help somewhere anywhere, I would be more than happy to do that! Just tag me
Cheers

**Pull request merged**